### DISCUSSION TITLE
CNAME Record

### DISCUSSION THREAD

**Discussion by edwarddarrah** [outline/outline#5558] [General] [answered]
Hello, 
My colleague was trying to update our domain. When he removed it from our Outline page, it broke our connection. Can we add the original Outline CNAME record for our account? If so, where can I find the record to re-add?

Thanks.

**Comment by tommoor** [comment, marked as answer]
If you're referring to the cloud-hosted custom domain feature the CNAME is `secure.getoutline.com`

**Reply by edwarddarrah** [reply]
Yes, it was for the cloud host. Thank you for sharing it.

**Comment by Codinator007** [comment]
Hi, self hoster here. Is there a way to do CNAME ing in self hosted as well?
original instance of outline: longsubdomain.example.com 
i set a cname from short.example.com to longsubdomain.example.com but i get a "not resolved" error.

**Reply by tommoor** [reply]
Sure, but it's left as an exercise for the reader – there are many ways you can achieve this. Make sure the `URL` is configured to point to the external-facing url.

**Reply by Codinator007** [reply]
Thank you, we did a redirect from the webserver side.
